container_pipeline/models/pipeline.py

```diff
@@ -23,7 +23,7 @@ class Project(models.Model):
     service
     """
     name = models.CharField(max_length=200, unique=True, db_index=True)
-
+    target_file_link = models.CharField(max_length=350, default=None)
```

Comment:
We should not have this field in `Projects` table.
The right place for this field is in `Build` table. As `target_file_link` can be changed per build, and every field should reflect correct link for target file.

If you are concern retrieving the link given a project-name, the build table has ForeginKey to Projects table, and you can retrieve builds of given project in descending order to grab the link from latest build.

Reply:
Well, i would vote for that too, but there is one slight issue. The link might get invalidated, if the user does an update to, say the git-path. 

The only way i can think of getting around this right now, is to store a copy of the source dockerfile on our end, and im not sure we want to do that.

Reply:
> The link might get invalidated

I think that's fine. It gives the record of what Dockerfile (path) being used to build for a particular build. We should not maintain the Dockerfiles themselves, rather we need to maintain and record artifacts we used for builds.